### ISSUE TITLE
Intel Ethernet Connection (2) i219-LM driver installs, but the device does not switch to new driver

### ISSUE THREAD

**Issue by TANielsen** [MSEndpointMgr/ConfigMgr#235]
Title sort of says it all.  I'm curious what could be causing this.  There are no errors in the logs about it (ApplyDriverPackage.log and Install-Drivers.txt) both show the driver is "installed".

It says "'Driver package is up-to-date on all applicable targets." but it is not according to what is listed in device manager.

Device Driver version in Device Manager = 12.51.25.6
Device Driver version that it installs = 12.18.9.10

It's even in the list of "show compatible hardware" but what is stumping me is why the device does not automatically switch to use the new driver?!

**Comment by TANielsen** [comment]
![i219-lm](https://user-images.githubusercontent.com/66029481/83064265-83997200-a02f-11ea-972c-63eb6c1abd6c.png)
Example of what I'm talking about

**Comment by Gordy01010111** [comment]
Yes its something specific about the Intel i219-LM driver. If you Check release notes for the driver it states you need to uninstall the old driver first before installing the new driver. you need to programmatically determine the OEM driver version using something like wmi pass the variable and then use something like pnputil.exe /delete-driver <oem#.inf> [/uninstall] [/force]. then when you go to install it it works then.
